microsoft-edge/devtools-guide-chromium/whats-new/2023/07/devtools-115.md

```diff
@@ -0,0 +1,129 @@
+---
+title: What's new in DevTools (Microsoft Edge 115)
+description: "Improvements for connecting DevTools to Azure Artifacts symbol server. Snapshotting improvements for the Memory tool. Refreshing the DevTools icons and buttons. And more."
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 08/18/2023
+---
+# What's New in DevTools (Microsoft Edge 115)
+
+[!INCLUDE [Microsoft Edge team note for top of What's New](../../includes/edge-whats-new-note.md)]
+
+
+<!-- ====================================================================== -->
+## Improvements for connecting DevTools to Azure Artifacts symbol server
+
+<!-- Subtitle: You can now use Azure Active Directory for authentication instead of Personal Access Tokens (PATs). You can connect to multiple symbol servers and filter which sourcemaps you want DevTools to fetch. -->
+
+In Microsoft Edge 115, it's easier and more customizable to connect DevTools to Azure Artifacts symbol server.  In previous versions of Microsoft Edge, to authenticate to the symbol server, you had to generate Personal Access Tokens (PATs) in Azure DevOps and then paste them into DevTools. Now, as long as you're logged into Microsoft Edge, you can connect to the symbol server by using Azure Active Directory.
+
+Additionally, you can now connect to multiple symbol servers, in case you need to fetch sourcemaps from different Azure DevOps organizations. You can also filter which sourcemaps you want DevTools to fetch from the symbol server, by specifying the sourcemap URLs in an inclusion or exclusion list.
```

Comment:
It might be worthwhile to specify what the wildcards match:

>
>  * `?` will match a single character
>  * `*` will match one or more of any character
> 
> "Inclusion list" mode will only attempt to look up source maps for scripts with URLs that match one of the entries in the list. "Exclusion list" mode will attempt to look up any source maps for scripts *except* those with URLs that match one of the entries in the list.
>
> An example might be to do something like:
> ```https://cdn.contoso.com/*```
> ```https://packages.contoso.com/*```
> on an Inclusion list will only attempt to resolve source maps matching these two URL patterns, and won't attempt to load source maps for other scripts.
>
> The Source Maps Monitor tool has been updated to show what has or has not been attempted.

Reply:
@robpaveza - in my Draft 2, i wish to be more specific than:
* "it attempts" (what is "it"?), 
* "it only attempts" (what is "it")? and 
* "what hasn't been attempted"

Draft 2:

In the **Filter behavior** text box, the following wildcards are supported:

| Wildcard | Meaning |
|---|---|
| ? | Matches a single character. |
| * | Matches one or more of any character. |

If you select **Exclusion List**, it attempts to look up any source maps for scripts except those that have a URL that matches one of the entries in the list.

If you select **Inclusion List**, it only attempts to look up source maps for scripts that have URLs that match one of the entries in the list.  For example, suppose you select **Inclusion List** in the **Filter behavior** dropdown list, and then enter the following in the **Filter behavior** text box:

```http
https://cdn.contoso.com/*
https://packages.contoso.com/*
```

In this example, it only attempts to resolve source maps that match these two URL patterns, and doesn't attempt to load source maps for other scripts.  The [Source Maps Monitor tool](../../../source-maps-monitor/source-maps-monitor-tool.md) has been updated to show what has or hasn't been attempted.

Reply:
i pushed that commit and entered fresh comments to fine-tune it